tests/integration/test_fim/test_files/test_audit/test_audit_after_initial_scan.py

```diff
@@ -9,12 +9,13 @@
 
 import pytest
 from wazuh_testing.fim import (LOG_FILE_PATH,
-                               callback_audit_reloaded_rule,
+                               callback_audit_added_rule,
                                callback_audit_removed_rule,
                                callback_audit_connection_close,
-                               callback_audit_connection)
+                               callback_audit_connection, wait_for_audit)
```

Comment:
There's a better way to do this when the import lines grow too much. You can import the module as follows:
```python
import wazuh_testing.fim as fim
```
and then, use `fim.LOG_FILE_PATH`, `fim.callback_audit_added_rule`